test/crossfilter-test.js

```diff
@@ -177,6 +176,46 @@ suite.addBatch({
       }
     },
 
+    "up to 64 dimensions supported": function() {
+      var data = crossfilter([]);
+      for (var i = 0; i < 64; i++) data.dimension(function() { return 0; });
+    },
+
+    "can add and remove 64 dimensions repeatedly": function() {
+      var data = crossfilter([]),
+          dimensions = [];
+      for (var j = 0; j < 10; j++) {
+        for (var i = 0; i < 64; i++) dimensions.push(data.dimension(function() { return 0; }));
+        while (dimensions.length) dimensions.pop().remove();
+      }
+    },
+
+    "filtering with more than 32 dimensions": function() {
+      var data = crossfilter();
+      var dims = {};
+
+      for (var i = 0; i < 50; i++) {
+        data.add([{value: i}]);
+      }
+
+      var dimfunc = function(i) {
+        return function(val) {
+          return val.value == i;
+        }
+      }
```

Comment:
It is, but since `true == 1` the test works ok. It just a boolean dimension. It then filters to 1 on each dimension and removes it, verifying that one record is removed each time this happens. It's a kind of in-exact filter test, but it does test the filtering on each of the 50 dimensions.